helm/values.yaml

```diff
@@ -0,0 +1,53 @@
+
+##Values to be provided to the deploment
+deployment:
+  labels:
+  ##Any labels required for the pods
+
+##Image specs
+image:
+  ui:
+    repository: octarinesec/kubescan-scanner-ui
+    tag: "20.0"
```

Comment:
20.2
We'll add this to next version.
If you need to run tests, you can type in your local clone "20.1.1"
Same for the second image.